### PR TITLE
Trying to fix several test issues

### PR DESCRIPTION
**Pull request by ronanwatkins**

### CODE REVIEW

File: genesyscloud/resource_genesyscloud_telephony_providers_edges_site_test.go [hunk 1]
```diff
@@ -501,7 +501,7 @@ func deleteLocationWithNumber(emergencyNumber string) error {
 						return err
 					}
 					_, err = locationsAPI.DeleteLocation(*location.Id)
-					time.Sleep(8 * time.Second)
+					time.Sleep(30 * time.Second)
```
Comment (carnellj) [inline genesyscloud/resource_genesyscloud_telephony_providers_edges_site_test.go:504]: See earlier comments

File: genesyscloud/resource_genesyscloud_telephony_providers_edges_site.go [hunk 2]
```diff
@@ -307,7 +313,7 @@ func readSite(ctx context.Context, d *schema.ResourceData, meta interface{}) dia
 	edgesAPI := platformclientv2.NewTelephonyProvidersEdgeApiWithConfig(sdkConfig)
 
 	log.Printf("Reading site %s", d.Id())
-	return withRetriesForRead(ctx, 90*time.Second, d, func() *resource.RetryError {
+	return withRetriesForRead(ctx, 180*time.Second, d, func() *resource.RetryError {
```
[the remark this answers is not in the document]
Comment (ronanwatkins) [reply genesyscloud/resource_genesyscloud_telephony_providers_edges_site.go:316]: I'll remove this and create a ticket. Even this wait isn't long enough, I'm still getting 404s when reading sites

File: genesyscloud/resource_genesyscloud_telephony_providers_edges_site_test.go (hunk 1 above)
Comment (ronanwatkins) [reply genesyscloud/resource_genesyscloud_telephony_providers_edges_site_test.go:504]: I think this one can be left in. It's a cleanup function that rarely runs

File: genesyscloud/data_source_genesyscloud_routing_email_domain_test.go [hunk 3]
```diff
@@ -22,11 +23,17 @@ func cleanupRoutingEmailDomains() {
...
+	shouldSleep := false
 	for _, routingEmailDomain := range *routingEmailDomains.Entities {
 		if routingEmailDomain.Id != nil && strings.HasPrefix(*routingEmailDomain.Id, "terraform") {
 			routingAPI.DeleteRoutingEmailDomain(*routingEmailDomain.Id)
+			shouldSleep = true
 		}
 	}
+
+	if shouldSleep {
+		time.Sleep(30 * time.Second)
```
[the remark this answers is not in the document]
Comment (ronanwatkins) [reply genesyscloud/data_source_genesyscloud_routing_email_domain_test.go:35]: I'll remove this and grab a correlation ID and create a ticket if it causes trouble in the future

**Pull request merged**